eng/native/configurecompiler.cmake

```diff
@@ -430,6 +430,8 @@ if (CLR_CMAKE_HOST_UNIX)
   # optimization in the IEEE 754 specification and is therefore considered unsafe.
   add_compile_options(-ffp-contract=off)
 
+  add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-fno-rtti>)
```

Comment:
I don't think we use this on Windows either. Can we try disabling there too?

Reply:
It is disabled on Windows: https://github.com/dotnet/runtime/blob/abfd53a204547b862b324ef5012d364916860c3c/eng/native/configurecompiler.cmake#L628-L631 IIRC, this replacement approach prevented a warning (related to `/GR` overriding `/GR-` or the other way around).